pytest.ini

```diff
@@ -2,4 +2,5 @@
 python_files = test_*.py
 testpaths =
     src/hupper
-    tests
\ No newline at end of file
+    tests
+filterwarnings = error
```

Comment:
this is so pytest will fail tests that issue https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest.PytestUnhandledThreadExceptionWarning